src/main/java/cz/muni/fi/pa165/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,40 @@
+package cz.muni.fi.pa165.dao;
+
+import java.util.List;
+
+import javax.inject.Named;
+import javax.persistence.*;
+
+import org.springframework.stereotype.Repository;
+
+import cz.muni.fi.pa165.entity.User;
+
+@Repository
+@Named
+public class UserDaoImpl implements UserDao {
+    @PersistenceContext
+    private EntityManager em;
+
+    public void create(User user) {
```

Comment:
Tady ta signatura sedí, jen neodpovídá interface, což by se ukázalo, pokud bys před implementaci metody dal anotaci @Override - hodilo by to chybu